all.json

```diff
@@ -1,6 +1,26 @@
 {
 	"allow": [],
 	"deny": [
+		"all-in-oneconnect.com",
+		"syncnodelist.com",
+		"secured.all-in-oneconnect.com",
+		"coin-claim.com",
+		"colablond.cc",
+		"collaplanb.cc",
+		"collablawd.cc",
+		"ragavio.ro",
```

Comment:
This name looks less problematic, cannot quite verify since the site itself is down.